### DISCUSSION TITLE
react h5端使用加载路径问题

### DISCUSSION THREAD

**Discussion by HydenLiu** [Tencent/libpag#364] [Q&A] [answered]
#### 问题

加载pag文件时，必须把文件放在public文件里面吗，如果我放到src里面的某个目录下，pag文件加载不出来，浏览器sources查看也没有该文件

#### 重现
代码使用的是你们的react damo，把文件放入src里面使用，就找不到路径了，404

#### 环境

libpag 版本：libpag(v4.0.5-release.11)  
设备：mac book pro (chrome 版本 102.0.5005.61（正式版本） (arm64))

**Comment by zenoslin** [comment, marked as answer]
放到public目录下只是为了打包的时候，可以把*.pag文件带上，做了一个示例。
如果你希望放到src目录下，需要配置好打包工具把*.pag文件当成资源文件打包到最终产物中。
*.pag文件是一个二进制文件，可以当成一个txt文件去进行http网络加载成arrayBuffer就可以了。
